deploy/bz2068601/10-etcdDivergentRevisionsSRE.PrometheusRule.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: custom-etcd-prometheus-rules
+  namespace: openshift-etcd-operator
+spec:
+  groups:
+  - name: etcd
+    rules:
+    - alert: etcdDivergentRevisionsSRE
+      annotations:
+        message: etcd is reporting divergent member revisions,
+          with a standard deviation of {{ $value }}.  Members may be
+          stuck or, in extreme circumstances, split brained.
+      expr: min_over_time(stddev(max by (pod) (etcd_debugging_mvcc_compact_revision{namespace="openshift-etcd"}))[5m:15s]) > 0
```

Comment:
Can we split this into a recording rule and then alert if it's `> 0`?

syntax example: https://github.com/openshift/managed-cluster-config/blob/master/deploy/sre-prometheus/100-control-plane-resizing.PrometheusRule.yaml